pkg/annotation/flatcar.go

```diff
@@ -0,0 +1,11 @@
+package annotation
+
+// support:
+//   - crd: awsmachinedeployments.infrastructure.giantswarm.io
+//     apiversion: v1alpha3
+//     release: Since 19.1.0
+//
+// documentation:
+//
+//	This annotation is used for selecting a specific flatcar release version for machine deployments.
+const FlatcarReleaseVersion = "alpha.flatcar.giantswarm.io/release-version"
```

Comment:
What is the expected value of this annotation? The channel name (stable, beta, etc) or the version number? If it's the version, how can we distinguish between alpha, beta and stable versions? Do we need to?

Reply:
version number, because this is how the lookup in aws-operator works.

in aws-operator PR I check the least minimum version 
https://github.com/giantswarm/aws-operator/pull/3593

Reply:
the value of this annotations confuses me a little bit.
I would maybe name it:

```
alpha.giantswarm.io/flatcar-release-version
```

to clearly distinguish between vintage platform release and flatcar specific release versions.

but not blocking, just my 2cc

Reply:
Changed and thanks for the suggestion 👍🏻